exec/java-exec/src/test/java/org/apache/drill/exec/store/pcap/TestPcapEVFReader.java

```diff
@@ -99,4 +99,10 @@ public void testAggregateQuery() throws Exception {
       .baselineValues(true, 16L)
       .go();
   }
+
+  @Test
+  public void testArpPcapFile() throws Exception {
+    String sql = "SELECT type AS packet_type FROM cp.`store/pcap/test2.pcap`";
+    queryBuilder().sql(sql).run();
```

Comment:
Please validate the query results. Also, the filename may be more descriptive to help further contributors to reuse it.

Reply:
Fixed.

Reply:
I think review comment was not addressed, `queryBuilder().sql(sql).run();` is still left...

Reply:
Oops... sorry about that.   I removed that.